internal/plugin/manager/manager_test.go

```diff
@@ -1,332 +0,0 @@
-package manager
```

Comment:
Why we have removed all the tests?

Reply:
The internal/plugin package has been deleted now by this [commit](https://github.com/notaryproject/notation-go/commit/2573c88a5f62cf8c637bd43aceb8fb255c0e82c9).